terraform/aws/components/pss/variables.tf

```diff
@@ -172,3 +172,32 @@ variable "pss_gp2gp_translator_container_port" {
   description = "Port Number on which service within container will be listening"
   default = 8085
 }
+
+variable "daisy_chaining_active" {
+  type = bool
+  description = "Is daisy chaining (gp2gp adaptor message forwarding) enabled"
+  default = false
+}
+
+variable "secret_name_sds_apikey" {
+  type = string
+  description = "SDS API Key"
+}
+
+variable "supported_file_types" {
+  type = string
+  description = "The attachment MIME types supported"
+  default = "application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document,application/msoutlook,text/rtf,text/plain,image/bmp,image/gif,image/jpeg,image/png,image/tiff,application/xml,audio/x-aiff,audio/x-mpegurl,audio/mpeg,audio/x-wav,audio/x-ms-wma,video/3gpp2,video/3gpp,video/x-ms-asf,video/x-ms-asf,video/x-msvideo,video/x-flv,video/quicktime,video/mp4,video/mpeg,audio/vnd.rn-realaudio,application/x-shockwave-flash,video/x-ms-vob,video/x-ms-wmv,application/postscript,application/postscript,image/svg+xml,image/x-pict,application/pdf,application/vnd.openxmlformats-package.relationships+xml,text/css,text/html,application/xhtml+xml,text/plain,application/json,text/xml,application/xml,application/pdf,audio/basic,audio/mpeg,image/png,image/gif,image/jpeg,image/tiff,video/mpeg,application/msword,application/octet-stream,text/csv,application/dicom,application/zip,application/x-rar-compressed,application/x-gzip,application/vnd.ms-excel,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet,application/x-mplayer2,audio/x-au,application/x-troff-msvideo,video/msvideo,image/x-windows-bmp,application/pkix-cert,application/x-x509-ca-cert,application/cdf,application/x-cdf,application/x-netcdf,application/x-x509-user-cert,application/EDIFACT,application/EDI-X12,application/EDI-Consent,application/hl7-v2,application/hl7-v2+xml,video/x-mpeg,application/pkcs10,application/x-pkcs10,application/pkcs-12,application/x-pkcs12,application/x-pkcs7-signature,application/pkcs7-mime,application/x-pkcs7-mime,application/pkcs7-mime,application/x-pkcs7-mime,application/x-pkcs7-certreqresp,application/pkcs7-signature,application/x-rtf,application/x-compressed,application/x-zip-compressed,multipart/x-zip,application/pgp,application/pgp-keys,application/pgp-signature,application/x-pgp-plugin,application/pgp-encrypted,audio/wav,audio/wave,audio/x-pn-wav,chemical/x-mdl-sdfile,chemical/x-mdl-molfile,chemical/x-pdb,application/x-hl7"
+}
+
+variable "pss_attachment_bucket_retention_period" {
+  type = number
+  description = "Number of days objects will be retained in pss_attachment_storage_bucket"
+  default = 7
+}
+
+variable "pss_storage_type" {
+  type = string
+  description = "The storage type used for attachments"
+  default = "LocalMock"
```

Comment:
Minor: Seems a little strange, to make this configurable given we're implementing AWS specific infra.

Reply:
Agreed, I'll hardcode it to S3